src/main/java/eu/eutampieri/catacombs/ui/World.java

```diff
@@ -48,12 +42,16 @@ public World(final TileMap tileMap, final DungeonGame game) {
 
         final SingleObjectFactory objectFactory = new SingleObjectFactoryImpl(this.tileMap);
         final Random rand = new Random();
-        this.entities.addAll(objectFactory.spawnSome(700, (x, y, tm) -> {
+        this.entities.addAll(objectFactory.spawnSome(5, (x, y, tm) -> {
```

Comment:
I think this is too low, maybe 20 would be neat

Reply:
maybe ... but if we make mobs drop potions i dont think we need 20 already on the ground